Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -11,9 +11,10 @@ import UIKit
 class NotificationsViewController: UIViewController {
     
     // MARK: - Views
-    
-    private let tableView = UITableView(frame: .zero, style: .insetGrouped)
+
     private let backButton = UIButton()
+    private let emptyStateView = UIView()
```

Comment:
This type of custom view should be factored out and be very flexible. In other words, you can pass in an image, title, and subtitle to this custom view. The reason is because this type of component is commonly used throughout the entire app's design system.